app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -112,25 +112,50 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
 <%= "          --data 'remote_execution_interface=#{@remote_execution_interface}' \\\n" if @remote_execution_interface.present? -%>
 <%= "          --data 'packages=#{@packages}' \\\n" if @packages.present? -%>
 
-}
+    }
 
     <% if @force -%>
-    yum remove -y katello-ca-consumer*
+    yum remove -y katello-ca-consumer* subscription-manager
     <% end -%>
 
-    CONSUMER_RPM=$(mktemp --suffix .rpm)
-    curl --silent --show-error --output $CONSUMER_RPM <%= subscription_manager_configuration_url(hostname: @url_host) %>
+    KATELLO_SERVER_CA_CERT=/etc/rhsm/ca/<%= @url.host %>.pem
```

Comment:
Currently, this file is:

```
/etc/rhsm/ca/katello-server-ca.crt
```

I think that mostly only matters due to how many users are used to this convention. I also think that naming the CA certificate after the host could be confusing to users. I'd vote stick with the way it is today `katello-server-ca.crt` or let's aim for some new naming convention that we want to aim for across the ecosystem (e.g. foreman_ca.crt, <hostname>-ca.crt)